seq2science/rules/configuration_generic.smk

```diff
@@ -474,7 +476,7 @@ wildcard_constraints:
 
 
 # make sure the snakemake version corresponds to version in environment
-min_version("6.12.3")
```

Comment:
since we'd need an exact version again:

```
import snakemake
from snakemake.exceptions import WorkflowError

if snakemake.__version__ != "5.18.1":
  raise WorkflowError(
    f"Expecting Snakemake version 5.18.1 (you are currently using {snakemake.__version__}
  )
```